label-syncer.yaml

```diff
@@ -0,0 +1,91 @@
+---
+org: "chainguard-dev"
+general:
+  - name: "bug"
+    color: "d73a4a"
+    description: "Something isn't working"
+  - name: "documentation"
+    color: "0075ca"
+    description: "Improvements or additions to documentation"
+  - name: "duplicate"
+    color: "cfd3d7"
+    description: "This issue or pull request already exists"
+  - name: "enforce"
+    color: "5F3992"
+    description: "Issue is associated to Chainguard Enforce"
+  - name: "enhancement"
+    color: "a2eeef"
+    description: "New feature or request"
+  - name: "good first issue"
+    color: "7057ff"
+    description: "Good for newcomers"
+  - name: "help wanted"
+    color: "008672"
+    description: "Extra attention is needed"
+  - name: "images"
+    color: "EB7CE7"
+    description: "Issue is associated to Chainguard Images"
+  - name: "invalid"
+    color: "e4e669"
+    description: "This doesn't seem right"
+  - name: "needs triage"
+    color: "5319E7"
+    description: ""
+  - name: "P0"
+    color: "B60205"
+    description: ""
+  - name: "P1"
+    color: "E99695"
+    description: ""
+  - name: "P2"
+    color: "FBCA04"
+    description: ""
```

Comment:
It would be nice if the description for these labels corresponded to internal expectations (response time, what it means, etc)